modules/test/playwright/tests/frontend-data-set-views-web/pages/view/visualization_modes/VisualizationModesPage.ts

```diff
@@ -33,14 +58,56 @@ export class VisualizationModesPage {
 		await this.viewPage.selectTab('Visualization Modes');
 	}
 
+	async openAssignFieldModal({
+		container,
+		sectionLabel,
+	}: {
+		container: Locator;
+		sectionLabel: string;
+	}) {
+		await container
+			.locator('tr')
+			.filter({has: this.page.getByText(sectionLabel)})
+			.getByTitle('Assign Field')
+			.click();
```

Comment:
I merged `list` and `cards` tests, pages and fixures into `visualizationModes` tests, pages and fixures. There is a lot of overlap between them, a lot of code that can be reused. But, we now need `container` prop to isolate locator withing a tab. Visualization mode tabs are all rendered, with only one visible. That makes is a bit more awkward to work with locators, as we need to find unique elements within all three tabs. That's the main reason for explicit containers. Sidenote - modal is only ever one, that's why we can have one modal locator, and not three.